src/qute/languageServer/client.ts

```diff
@@ -76,11 +77,22 @@ export function connectToQuteLS(context: ExtensionContext) {
       bindQuteRequest('qute/java/diagnostics');
       bindQuteRequest('qute/java/documentLink');
       bindQuteNotification('qute/dataModelChanged');
+
+      // When an editor is changed, check the qute validation context.
+      context.subscriptions.push(
+        window.onDidChangeActiveTextEditor(async editor => {
+          await updateQuteContext(editor.document);
```

Comment:
I noticed when closing the last opened file, the active text editor is set to null, and this callback does get informed. This produces :

```
stack trace: TypeError: Cannot read property 'document' of undefined
	at /home/rgrunber/git/vscode-quarkus/dist/extension.js:85792:72
 ...
 ...
```

I don't think it affects anything but it does generate the error.

Reply:
I think @AlexXuChen have fixed that. If it's true @AlexXuChen could you `Resolve conversation` please.